nucliadb_models/src/nucliadb_models/writer.py

```diff
@@ -94,14 +94,25 @@ class CreateResourcePayload(BaseModel):
         title="Origin",
         description="Origin metadata for the resource. Used to store information about the resource on the origin system. Most of its fields can later be used to filter at search time.",  # noqa
     )
-    extra: Optional[Extra] = None
-    hidden: Optional[bool] = None
+    extra: Optional[Extra] = Field(
+        default=None,
+        title="Extra",
+        description="Extra metadata for the resource. It can be used to store structured information about the resource that can't be used to query at retrieval time.",
+    )
+    hidden: Optional[bool] = Field(
+        default=None,
+        title="Hidden",
+        description="Set the hidden status of the resource. If not set, the default value for new resources in the KnowledgeBox will be used.",
+    )
 
     files: Dict[FieldIdString, FileField] = FieldDefaults.files
     links: Dict[FieldIdString, LinkField] = FieldDefaults.links
     texts: Dict[FieldIdString, TextField] = FieldDefaults.texts
     conversations: Dict[FieldIdString, InputConversationField] = FieldDefaults.conversations
-    processing_options: Optional[PushProcessingOptions] = PushProcessingOptions()
+    processing_options: Optional[PushProcessingOptions] = Field(
```

Comment:
Out of curiosity: Will this field be marked as deprecated at some point (using extract strategy instead)?

Reply:
I think so, but we have to discuss with learning